src/renderer/i18n/en/wallet.ts

```diff
@@ -11,6 +11,9 @@ const wallet: WalletMessages = {
   'wallet.action.upgrade': 'upgrade',
   'wallet.action.receive': 'Receive {asset}',
   'wallet.action.remove': 'Remove wallet',
+  'wallet.action.remove.title': 'Are you sure you want to remove your wallet?',
+  'wallet.action.remove.description':
+    'You cannot undo this action and will have to re-create your wallet from your seed phrase.',
```

Comment:
'wallet.action.remove.description':
    'To o re-create your wallet you need your phrase. Please make sure you have it already saved at a safe place.',

Reply:
@veado as English being my primary language, I think the current copy I have is more clear. Perhaps something about it is confusing and I can try to adjust it, which part do you find confusing?

Reply:
> as English being my primary language, I think the current copy I have is more clear

@warrenshen Agree :smile: The confusing part for me was just "You cannot undo this action". From my perspective any user can still undo this action by re-importing the wallet, but he/she have to make sure to backup the phrase before (which is needed for the import).

Reply:
Makes sense, I can see how that is confusing. How about this?

"You will need to provide your seed phrase to re-create your wallet. Please ensure you have your seed phrase saved somewhere before you proceed."

Reply:
Nice! Just small change (we don't use `seed phrase`, just `phrase` and `somewhere` -> `at a safe place`:
```
You will need to provide your phrase to re-create your wallet. Please ensure you have your phrase saved at a safe place before you proceed.
```

We are going to add such change right now to merge your PR, so we can bring it into next release.

Reply:
Thanks for the change @veado